beams/setup.py

```diff
@@ -799,12 +922,336 @@ def get_job_applicant_custom_fields():
                 "options": "Location",
                 "insert_after": "country"
+            },
+            {
+                "fieldname": "current_employer_tab_break",
+                "fieldtype": "Tab Break",
+                "label": "Current Employer Details",
+                "insert_after": "upper_range"
+            },
+            {
+                "fieldname": "current_employer",
+                "fieldtype": "Section Break",
+                "label": "Current Employer / Immediate Previous Employer",
+                "insert_after": "current_employer_tab_break"
+            },
+            {
+                "fieldname": "name_of_employer",
+                "fieldtype": "Data",
+                "label": "Name of Employer",
+                "insert_after": "current_employer"
+            },
+            {
+                "fieldname": "employee_code",
+                "fieldtype": "Int",
+                "label": "Employee Code",
+                "insert_after": "name_of_employer"
+            },
+            {
+                "fieldname": "telephone_no",
+                "fieldtype": "Int",
+                "label": "Telephone No",
+                "insert_after": "employee_code"
+            },
+            {
+                "fieldname": "employment_period_from",
+                "fieldtype": "Int",
+                "label": "Employment Period From",
+                "insert_after": "telephone_no"
+            },
+            {
+                "fieldname": "employment_period_to",
+                "fieldtype": "Int",
+                "label": "Employment Period To",
+                "insert_after": "employment_period_from"
+            },
+            {
+                "fieldname": "address_of_employer",
+                "fieldtype": "Small Text",
+                "label": "Address of Employer",
+                "insert_after": "employment_period_to"
+            },
+
+            {
+                "fieldname": "current_employer_1_column_break",
+                "fieldtype": "Column Break",
+                "label": "",
+                "insert_after": "address_of_employer"
+            },
+
+            {
+                "fieldname": "first_salary_drawn",
+                "fieldtype": "Float",
+                "label": "First Salary Drawn",
+                "insert_after": "current_employer_1_column_break"
+            },
+            {
+                "fieldname": "last_salary_drawn",
+                "fieldtype": "Float",
+                "label": "Last Salary Drawn",
+                "insert_after": "first_salary_drawn"
+            },
+            {
+                "fieldname": "current_designation",
+                "fieldtype": "Data",
+                "label": "Designation",
+                "insert_after": "last_salary_drawn"
+            },
+            {
+                "fieldname": "reference_taken",
+                "fieldtype": "Select",
+                "label": "Can a reference taken now?",
```

Comment:
"Can a reference be taken now?" <- Not sure about this, reflect on the context and decide on a better label